extractor/src/main.py

```diff
@@ -187,10 +187,21 @@ def broadcast_events(dataframe, analysis, batch_id=None):
                 event["resource_type"] = resource_type
                 event["resource_id"] = resource_id
                 event["record"] = record
-                producer.produce_event(
-                    topic=f"{PRODUCED_TOPIC_PREFIX}{batch_id}",
-                    event=event
-                )
+                try:
+                    producer.produce_event(
+                        topic=f"{PRODUCED_TOPIC_PREFIX}{batch_id}",
+                        event=event
+                    )
+                except KafkaException as e:
+                    if e.args[0].code() == KafkaError.UNKNOWN_TOPIC_OR_PART:
+                        logger.debug(f"extract topic for batch {batch_id} does not exist")
```

Comment:
If we're sure that it's because the topic was deleted, could we add this reason in the message?